job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/ScheduledExecutor.java

```diff
@@ -15,21 +15,20 @@
  */
 package com.hpe.caf.services.job.scheduled.executor;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import java.util.Arrays;
 import java.util.concurrent.Executors;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
Imports should not be moved except in a commit expressly for the purpose for organizing them.

Reply:
Reverted